ticktock-android/tzdb/build.gradle

```diff
@@ -19,13 +19,20 @@ plugins {
   id 'dev.zacsweers.ticktock'
 }
 
+android {
+  compileOptions {
+    coreLibraryDesugaringEnabled true
+  }
+}
```

Comment:
My understanding is that this shouldn't be applied on library projects

Reply:
Lint was failing without because I call `Supplier.get()` in here

Reply:
I can also ignore the lint

Reply:
Yeah I suspect that’s a false positive because of the set minSdk version. Ignoring sounds good 👍